src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -83,17 +83,20 @@ DateTimeOffset TimeStamp
 
 public record Node
 (
-    DateTimeOffset? InitializedAt,
     [PartitionKey] PoolName PoolName,
-    Guid? PoolId,
     [RowKey] Guid MachineId,
-    NodeState State,
-    Guid? ScalesetId,
-    DateTimeOffset Heartbeat,
-    string Version,
-    bool ReimageRequested,
-    bool DeleteRequested,
-    bool DebugKeepNode
+    Guid? PoolId,
+    Version Version,
```

Comment:
This won't work. The version class does not support the format of custom version we use when we deploy from  git. 
example "5.3.0-fe0ae308234cb6e5e39193e20f5819108ff78856"